internal/pkg/dsl/node.go

```diff
@@ -33,6 +33,14 @@ func (n *Node) MarshalJSON() ([]byte, error) {
 	return []byte(kKeywordNULL), nil
 }
 
+func (n *Node) MustMarshalJSON() []byte {
+	res, err := n.MarshalJSON()
+	if err != nil {
+		panic(err)
```

Comment:
Panic sounds extreme here?

Reply:
This is the expected behavior of any `Must` function. It is only used in a global var, which would get caught with testing.

The usage of a `Must` function should always be cautions, but in the case it is used correctly.